test/manual/amp-lightbox-viewer-optin.amp.max.html

```diff
@@ -0,0 +1,127 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Lightbox Viewer (Opt-in)</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="../../dist/amp.js"></script>
+  <script async custom-element="amp-lightbox-viewer" src="../../dist/v0/amp-lightbox-viewer-0.1.max.js"></script>
+  <script async custom-element="amp-youtube" src="../../dist/v0/amp-youtube-0.1.max.js"></script>
+  <script async custom-element="amp-twitter" src="../../dist/v0/amp-twitter-0.1.max.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    body {
+      font-family: sans-serif;
+    }
+
+    h2, h3, h4 {
+      margin-top: 30px;
+      margin-bottom: 5px;
+      color: 333;
+      font-weight: normal;
+    }
+
+    ul {
+      margin: 10px;
+      padding: 0;
+      box-shadow: 0 0 10px #999;
+      box-sizing: border-box;
+    }
+
+    ul li {
+      padding: 10px;
+      border-top: 1px solid #ccc;
+      list-style-type: none;
+    }
+    ul li:first-child {
+      border-top: none;
+    }
+
+    ul.amp-lightboxed {
+      background-color: #fff;
+    }
+
+    blockquote {
+      background: #EDE7F6;
+      border-left: 10px solid #6A1B9A;
+      margin: 10px;
+      padding: 10px;
+      font-family: Georgia, serif;
+      font-size: 18px;
+      font-style: italic;
+    }
+
+    blockquote:before {
+      color: #FF4081;
+      content: open-quote;
+      font-size: 36pt;
+    }
+
+    .create-new-stacking-context {
+      position: relative;
+      z-index: 10;
+    }
+  </style>
+</head>
+<body>
+  <script>
+    setTimeout(function() {
+      if (!AMP.isExperimentOn('amp-lightbox-viewer') || AMP.isExperimentOn('amp-lightbox-viewer-auto')) {
+        if (!AMP.isExperimentOn('amp-lightbox-viewer')) {
+          AMP.toggleExperiment('amp-lightbox-viewer');
+        }
+        if (AMP.isExperimentOn('amp-lightbox-viewer-auto')) {
```

Comment:
just automatically enabling experiments instead of a button to do it. added some comments.